app/components/checkbox/checkbox.css

```diff
@@ -14,3 +14,52 @@
   flex-grow: 1;
   flex-shrink: 1;
 }
+
```

Comment:
Just curious, is there any particular reason for using a switch-style checkbox instead of just a default one?

Reply:
No, no real motivation for using a switch. I used a checkbox first and was unhappy with its look. Since @drjakob likes switches, I went for a switch.